### PR TITLE
Add docblock for methods when implements

### PR DESCRIPTION
**Pull request by ptcong**
- Remove "abstract" keyword when implements
- Copy docblock from interface, abstract methods to impelmented class to make it more clear.

### CODE REVIEW

File: php_companion/commands/parse.py [hunk 1]
```diff
@@ -56,12 +61,14 @@ def on_done(self, index):
 
         # Better way to handle add all selection?
         if index == 0:
+            methods = ""
```
Comment (ptcong) [inline php_companion/commands/parse.py]: to undo faster if needed

**Comment by erichard** [comment]
Look great ! :+1: for the inherit option 

Thanks

**Comment by ptcong** [comment]
@erichard If it's ok, should merge it.

**Comment by ptcong** [comment]
Thanks

**Pull request merged**